generators/bdk/python/templates/ext-app/main.py.ejs

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+import os
+import ssl
+
+from aiohttp import web
+from symphony.bdk.core.config.loader import BdkConfigLoader
+
+from python.ext_app_be import CircleOfTrustService, CircleOfTrustHandler, APP_BASE_PATH
+
+CURRENT_DIR = os.path.dirname(os.path.abspath(__file__))
+RESOURCES_FOLDER = os.path.join(CURRENT_DIR, "../resources")
+STATIC_FOLDER = os.path.join(RESOURCES_FOLDER, "static")
+
+def run_app():
+    ssl_context = ssl.create_default_context(ssl.Purpose.CLIENT_AUTH)
+    # pass the server SSL certificate and private key
+    cert_file_path = os.path.join(STATIC_FOLDER, "srv.crt")
```

Comment:
those are autosigned certificates right? will it work out of the box everyone or will they have to authorize them or something locally?

Reply:
Yes these are self-signed certificate extracted from https://github.com/SymphonyPlatformSolutions/generator-symphony/blob/master/generators/bdk/java/templates/ext-app/resources/keystore.p12
I could add a comment saying certs are self-signed and need to be updated